examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
You'll have to be more specific what's causing your discomfort...

Here we had two bugs: one was improperly applying projectors in our PSD
functions. That seems right to fix here.

The other bug was notch filtering selected channels in an example, thinking
that would properly remove line noise even after projection. This bug was
hidden by the first one, so I think it should also be fixed here to make
the example work properly.

There might also be other bits of code that suffer from similar picking or
projection subselection errors, and I think those should be taken care of
in another PR.

Do you agree?